testing-internal/src/main/java/com/linecorp/armeria/internal/testing/InternalTestingBlockHoundIntegration.java

```diff
@@ -75,6 +75,7 @@ public void applyTo(Builder builder) {
                 "writeBlockingMethod");
         builder.allowBlockingCallsInside("com.linecorp.armeria.client.ClientFactory", "ofDefault");
         builder.allowBlockingCallsInside("io.envoyproxy.controlplane.cache.SimpleCache", "createWatch");
+        builder.allowBlockingCallsInside("io.grpc.netty.shaded.io.grpc.netty.GrpcHttp2ConnectionHandler", "<clinit>");
```

Comment:
`GrpcHttp2ConnectionHandler` changed to read Netty `Version` which is reported as a blocking operation.

https://github.com/grpc/grpc-java/pull/11367/files#diff-ac0ae9f2ac0baa6aad0372713c102af83ea972c6fed7ddcc2767e51dbb372f8fR55

```java
Thread[#42,grpc-default-worker-ELG-3-1,5,main]
java.lang.Exception: java.io.RandomAccessFile#readBytes
        at com.linecorp.armeria.internal.testing.InternalTestingBlockHoundIntegration.writeBlockingMethod(InternalTestingBlockHoundIntegration.java:85)
        at reactor.blockhound.BlockHound$Builder.lambda$install$8(BlockHound.java:488)
        at reactor.blockhound.BlockHoundRuntime.checkBlocking(BlockHoundRuntime.java:89)
        at java.base/java.io.RandomAccessFile.readBytes(RandomAccessFile.java:398)
        at java.base/java.io.RandomAccessFile.read(RandomAccessFile.java:434)
        at java.base/java.util.zip.ZipFile$Source.readAt(ZipFile.java:1541)
        at java.base/java.util.zip.ZipFile$ZipFileInputStream.read(ZipFile.java:956)
        at java.base/java.util.zip.ZipFile$ZipFileInflaterInputStream.fill(ZipFile.java:463)
        at java.base/java.util.zip.InflaterInputStream.read(InflaterInputStream.java:175)
        at java.base/java.io.FilterInputStream.read(FilterInputStream.java:119)
        at java.base/java.io.FilterInputStream.read(FilterInputStream.java:95)
        at java.base/java.util.Properties$LineReader.readLine(Properties.java:504)
        at java.base/java.util.Properties.load0(Properties.java:420)
        at java.base/java.util.Properties.load(Properties.java:409)
        at io.grpc.netty.shaded.io.netty.util.Version.identify(Version.java:77)
        at io.grpc.netty.shaded.io.netty.util.Version.identify(Version.java:56)
        at io.grpc.netty.shaded.io.grpc.netty.GrpcHttp2ConnectionHandler.<clinit>(GrpcHttp2ConnectionHandler.java:55)
        at io.grpc.netty.shaded.io.grpc.netty.NettyServerTransport.createHandler(NettyServerTransport.java:269)
        at io.grpc.netty.shaded.io.grpc.netty.NettyServerTransport.start(NettyServerTransport.java:137)
```